easybuild/easyconfigs/c/CHASE/CHASE-v1.eb

```diff
@@ -0,0 +1,28 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Ravi Tripathi
+# Email: ravi89@uab.edu
+
+easyblock = "Tarball"
+
+name = 'CHASE'
+version = 'v1'
```

Comment:
@boegel: It would. I just went with how it was earlier installed on our clusters. I'll push the changes.